fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Curious: is the leading newline significant?

Reply:
Nope! It's just as-is generated from `cargo fuzz init`.